libsawtooth/src/protocol/setting.rs

```diff
@@ -140,7 +140,7 @@ impl IntoBytes for Vec<Setting> {
 impl IntoNative<Vec<Setting>> for SettingProto {}
 impl IntoProto<SettingProto> for Vec<Setting> {}
 
-/// Builder for the `Setting` struct
+/// Builder for [`Setting`](struct.Setting.html)
```

Comment:
Looks like this change should be in the setting commit

Reply:
Fixed